src/libopensc/card-cardos.c

```diff
@@ -985,10 +1191,41 @@ cardos_decipher(struct sc_card *card,
 		const u8 * crgram, size_t crgram_len,
 		u8 * out, size_t outlen)
 {
+	cardos_data_t* priv = (cardos_data_t*)card->drv_data;
```

Comment:
clang-format suggested the following change:


```
-	cardos_data_t* priv = (cardos_data_t*)card->drv_data;
+	cardos_data_t *priv = (cardos_data_t *)card->drv_data;
```

<!-- identifier: clang-format -->
<!-- File: src/libopensc/card-cardos.c -->